### DISCUSSION TITLE
Alpha 165

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#5074] [Release Notes]
## Commits

- SSH: native SOCKSv5 connection support
- SSH: UI improvements to port forwarding (@highfredo) [#4959](https://github.com/Eugeny/tabby/pull/4959), [#4959](https://github.com/Eugeny/tabby/pull/4959)
- SSH: respect keyboard-interactive prompt echo setting requested by the server
- SSH: disabled blacklisted key exchange algorithms (set `TABBY_ENABLE_SSH_ALG_BLACKLIST` environment variable to re-enable)
- SSH: fixed X11 display spec parsing
- SSH: fixed WinSCP integration for profiles with no username set
- Fixed hotkey setter to handle backticks and tildes properly (Nessun Kim) [#4992](https://github.com/Eugeny/tabby/pull/4992)
- The clickable-links plugin is now bundled with Tabby
- Profile settings: allow filtering profiles by description 
- Fixed tab title recovery for split tabs

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.165'>Alpha 165</a>.</em>

**Comment by berserker5000** [comment]
Hi there, I've updated my MacOs to the latest one (Monterey), and your gorgeous terminal is not working on it :( Can you please do something with it? I don't want to use any other terminals :)

**Comment by zorrohahaha** [comment]
I downloaded the portable version for win10 and it cannot be loaded.

**Reply by Eugeny** [reply]
Fixed in v166

**Comment by TimaGribanov** [comment]
I have downloaded new version for Win10 and cannot run it, the process is being created, but nothing more happens

**Reply by Eugeny** [reply]
Fixed in v166
